packages/odata-v4/src/de-serializers/default-de-serializers.ts

```diff
@@ -14,6 +14,7 @@ import {
   serializeToTime,
   deserializeDateToMoment
 } from './converters';
+import type { Moment } from 'moment';
```

Comment:
I think this is not the correct way to import moment.

Reply:
I now removed the import as I am `moment` seems to be accessible without it in this file.

But, I don't understand how this works, and why we explicitly use `import { Moment } from 'moment'` in some places but use it directly in some.

Reply:
I think it's likely due to moment being a global namespace here (which allows using it without import statement). But for consistency we should still import it.

Reply:
If you're using moment only for its type then use:

import type moment else import moment